src/index.ts

```diff
@@ -48,6 +69,21 @@ const getLogin = async (octokit: Octokit): Promise<string> => {
   return login;
 }
 
+// If repoConfig (the file repositories.json) exists, read it and return the list of repositories from the repositories key
+let repoConfig = 'repositories.json'
+const getRepositoriesFromJson = async (): Promise<Repository[]> => {
+  const fs = await import('fs').then(m => m.default);
```

Comment:
nit: Usually you'd put the imports at the top of the file, rather than dynamically importing at this point. But I can see the argument for not trying to import them if you're not reading files.